packages/apps/api-authorizer/src/index.ts

```diff
@@ -1,277 +1,21 @@
-import Auth0Client, { Auth0UserInfo } from '@weco/auth0-client';
```

Comment:
Does this file need to continue to exist of should we fold it into handler.ts?

Reply:
It does - because the redis/auth0 services live at the module scope, importing it will initialise them (which throws an error in tests). Keeping it like this to export the "real" version of the handler links up with [here](https://github.com/wellcomecollection/identity/blob/ca609e1c475826889b578649972db8bb79833a53/infra/scoped/lambda.tf#L5)